Pages/Shared/_Layout.cshtml

```diff
@@ -3,14 +3,16 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline';">
```

Comment:
Helps prevent cross-site scripting (XSS) and other code injection attacks
Provides an additional layer of security for web application